internal/provider/config/config.go

```diff
@@ -47,12 +49,13 @@ type Config struct {
 }
 
 type KubeVIP struct {
-	Args        []string `yaml:"args,omitempty"`
-	EIP         string   `yaml:"eip,omitempty"`
-	ManifestURL string   `yaml:"manifest_url,omitempty"`
-	Interface   string   `yaml:"interface,omitempty"`
-	Enable      *bool    `yaml:"enable,omitempty"`
-	StaticPod   bool     `yaml:"static_pod,omitempty"`
+	EIP         string `yaml:"eip,omitempty"`
+	ManifestURL string `yaml:"manifest_url,omitempty"`
+	Interface   string `yaml:"interface,omitempty"`
+	Enable      *bool  `yaml:"enable,omitempty"`
+	StaticPod   bool   `yaml:"static_pod,omitempty"`
+	Version     string `yaml:"version,omitempty"`
+	kubevip.Config
```

Comment:
this reminds me that we haven't looked into the schema and validator in a while now 😅 maybe we should just remove it or change it to be based on the config only to avoid the drift

Reply:
another example of the drift: https://github.com/kairos-io/kairos/discussions/3342#discussioncomment-12969883